Global/DEBUG_VAR.gd

```diff
@@ -8,7 +8,12 @@ extends Node
 ## Player can move freely on the map without having to finish the current event or move in the range of its normal movement
 const DEBUG_FREE_MOVEMENT: bool = false
 
+## Allow player to add and remove items from the inventory with buttons [br]
+## in the inventory UI
+
+const DEBUG_ACTIVE_INVENTORY_DEBUG_BUTTONS : bool = true
+
 ## List of all debug variables [br]
 ## This is used to run the tests checking that every DEBUG is set to false
 ## ! If you add a new DEBUG variable, add it to this list
-const LIST_OF_DEBUG: Array[bool] = [DEBUG_FREE_MOVEMENT]
+const LIST_OF_DEBUG: Array[bool] = [DEBUG_FREE_MOVEMENT, DEBUG_ACTIVE_INVENTORY_DEBUG_BUTTONS]
```

Comment:
When you merge the main, remove this list at the end, the tests automatically generates the list, so it's not needed here anymore

Reply:
I leave removing the array to the sceneloader pr as main still has the array and that pr is the first I've seen to remove it, just in case it makes some complications right now

Reply:
oh you are right i remove it in my PR, it's not in the main, leave it like that